modules/mobile/spec/models/adapters/appointments_vaos_v2_adapter_spec.rb

```diff
@@ -5,11 +5,15 @@
 describe Mobile::V0::Adapters::VAOSV2Appointments, :aggregate_failures do
   # while hashes will work for these tests, this better reflects the data returned from the VAOS service
   def appointment_data(index = nil)
-    parsed = JSON.parse(appointment_fixtures, symbolize_names: true)
-    appts = index ? parsed[index] : parsed
+    appts = index ? raw_data[index] : raw_data
     Array.wrap(appts).map { |appt| OpenStruct.new(appt) }
   end
 
```

Comment:
I think this spec suite could use some cleanup. I initially wrote it as more of an integration style "test this broad case". That was a poor choice that hasn't grown well. I also wrote it to find appointments from the raw data by index, which was also a poor choice and led to further questionable decisions like having multiple appointments with the same id. I've been thinking about making a new ticket for cleaning this up a bit.

Reply:
Yeah pls make a new ticket for that.